src/it/java/com/uvocab/api/controller/UserControllerTest.java

```diff
@@ -20,9 +20,10 @@ class UserControllerTest {
 
   @Test
   void shouldPostTheUser() {
-    var user = User.newBuilder().setId(1).setLogin("danilo").setEmail("danilo@uvocab.edu").build();
+    var user = User.newBuilder().setLogin("danilo").setEmail("danilo@uvocab.edu").build();
     var response = restTemplate.exchange("/v1/user", POST, new HttpEntity<>(user), User.class);
     assertEquals(HttpStatus.OK, response.getStatusCode());
-    assertEquals(1, response.getBody().getId());
+    assertEquals("danilo", user.getLogin());
+    assertEquals("danilo@uvocab.edu", user.getEmail());
```

Comment:
@Danilosimoes we need assert the user id to validate the test "end to end"  controller -> service -> repo